torch/csrc/jit/README.md

```diff
@@ -0,0 +1,25 @@
+# PyTorch JIT
+
+This folder contains (most of) the C++ code for the PyTorch JIT, a language
+and compiler stack for executing PyTorch models portably and efficiently. To
+learn more about the JIT from a user perspective, please consult our
+[reference documentation](https://pytorch.org/docs/stable/jit.html) and
+[tutorials](https://pytorch.org/tutorials/beginner/Intro_to_TorchScript_tutorial.html).
+
+A brief summary of the source tree:
+- `OVERVIEW.md`: High-level technical overview of the JIT.
+- `frontend/`: Taking PyTorch modules in Python and translating them into the JIT IR.
+- `ir/`: Core IR abstractions.
+- `runtime/`: Interpreter, graph execution, and JIT operators.
+- `codegen/`: Generating efficient, hardware-specific code for JIT subgraphs.
+- `serialization/`: Saving and loading modules.
+- `api/`: Any user-facing C++ or Python interfaces.
+- `python/`: Binding stuff into Python or accessing information from the Python environment.
+- `testing/`: Utilities and helpers for testing.
+- `mobile/`: Mobile-specific implementations of runtime components.
+- `passes/`: IR-to-IR passes, generally for optimization and lowering.
```

Comment:
Can you add a bit about the existing `backends`, `generated`, `codegen` `tensorexpr`, and `testing` folders

Reply:
I think `codegen` and `testing` are already documented. I plan to move `tensorexpr` under `codegen`. And `backends` is to be deleted shortly. I'll add a note about `generated`.